app/models.py

```diff
@@ -2,6 +2,17 @@
 from sqlalchemy.dialects.postgresql import JSON
 
 
+class Frameworks(db.Model):
+    __tablename__ = 'frameworks'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    name = db.Column(db.String(255), nullable=False)
+
+    expired = db.Column(db.Boolean, index=False, unique=False,
```

Comment:
I wonder if this should be a `expired_at` datetime instead? That way we can just set start and end dates on frameworks in advance, without having to flip the switch manually.